src/components/Game.js

```diff
@@ -890,10 +903,14 @@ class Game extends React.Component {
       for (let i = 0; i < teamASteps; i++){
         this.playbook[i] = this.getRole(teams.A.length, teams.A.indexOf(userName), i);
       }
+      this.playbook[teamASteps + teamBSteps - 1] = "REPLAY";
```

Comment:
Can we handle this according to # members in team B?

Reply:
Fix it now. Let me know if you have any questions.